### PR TITLE
docs(vital-examples): Clean up Curriculum Lessons

### PR DESCRIPTION
**Pull request by misterjones**
## Changes

- Perform "doc clean-up" on Curriculum Lessons.
  - Fix typos, formatting, etc.
  - Replace applicable code blocks with embedded code from the associated example files.
- Update example files.
  - Restrict code comments to column 78 (for Docsify).

### CODE REVIEW

File: packages/vital-examples/doc/Curriculum/Lessons/DevelopANewComponent.md [hunk 1]
```diff
@@ -1,32 +1,68 @@
...
- - Creating a new Clean Vital Component from scratch.
- - Creating a set of Tokens for the clean vital component.
- - Composing and extending vital tokens at the Brand package and Site.
- - Token Naming Convention. `WithSectionLink` vs `SectionLink`.
- 
+- Creating a new _clean_ Vital component from scratch.
+- Creating a set of tokens for the clean Vital component.
+- Composing and extending Vital tokens at the brand package and site level.
+- The token naming convention (e.g., `WithSectionLink` vs `SectionLink`).
+
```
Comment (hvanyo) [inline packages/vital-examples/doc/Curriculum/Lessons/DevelopANewComponent.md:17]: @misterjones  here I would start with -- we suggest to make use of the [component scaffolder](http://localhost:3000/#/VitalDesignSystem/Guides/ComponentTemplate?id=component-scaffolding) to create this component structure automatically.  (give example of options they should give to create 'section' component) and then can make these changes to build the component.

File: packages/vital-examples/doc/Curriculum/Lessons/ReusableTokens.md [hunk 2]
```diff
@@ -2,22 +2,59 @@
...
+
+</div>
 
 ## Assignment
 
-### Creating a new CSS rule in Tailwind for our image border radius
+### Creating a New CSS Rule in Tailwind for Our Image Border Radius
 
-To begin, we'll use the `addComponents` function provided by Tailwind to first create a collection of CSS rules containing the corner radius styling to be applied to the 'Hero' card.
+To begin, we'll use the `addComponents` function provided by Tailwind to first create a collection
```
Comment (hvanyo) [inline packages/vital-examples/doc/Curriculum/Lessons/ReusableTokens.md:46]: @misterjones  for this one -- not really feeling this is great way to start this.  I wanted an example of creating a simple token and complex token.  We start of diving super deep into complex one.  Then show the simple one and I think takes away from this lesson.   

Can you restructure this that we start with "Create our reusable Radius Tokens"  then in that section point to "how to ceate fancy one"    Then follow the "Creating A Fancy Radius"  (I think chaning the name would help).  Then this is second section.

File: packages/vital-examples/doc/Curriculum/Lessons/DevelopANewComponent.md (hunk 1 above)
Comment (misterjones) [reply packages/vital-examples/doc/Curriculum/Lessons/DevelopANewComponent.md:17]: ✅ Updated.

@hvanyo, I recommended the use of the _Component Scaffolder_ and provided example instructions on using it.

See: [8a674c4](https://github.com/johnsonandjohnson/Bodiless-JS/pull/2137/commits/8a674c4f21d1cf896af3e0cb397c65a0975ef033)

File: packages/vital-examples/doc/Curriculum/Lessons/ReusableTokens.md (hunk 2 above)
Comment (misterjones) [reply packages/vital-examples/doc/Curriculum/Lessons/ReusableTokens.md]: ✅ Updated.

@hvanyo, based on your feedback, I:

- Rearranged the sections to progress from the `Simple` to the `Fancy` token.
  - This took some tweaking, but I believe it flows well.
- Moved some discussion out into its own "The Benefits of Tokens" section.

See: [`9d9080f`](https://github.com/johnsonandjohnson/Bodiless-JS/pull/2137/commits/9d9080f32b0ba9ea1b5ddb768e421502ca8d0073)

**Review by hvanyo — APPROVED** [review]

**Review by KaterinaPashkevich — APPROVED** [review]

**Pull request merged**